fidesctl/src/fidesctl/core/export.py

```diff
@@ -0,0 +1,219 @@
+"""This module handles the logic required for applying manifest files to the server."""
+from datetime import datetime
+from typing import Dict, List, Tuple, Set
+
+import csv
+
+from fidesctl.core.api_helpers import get_server_resources
+from fidesctl.core.utils import echo_green, get_all_level_fields
+
+
+def list_fides_keys(resources: List) -> List:
+    """
+    Returns a list of fides_keys found in the manifest dir
```

Comment:
`found in the resource list`

Reply:
deleted the function but great catch, thanks for clarifying that for my brain!